src/lib/components/LanguageSelect/index.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview A select for choosing language option.
+ */
+
+import {html} from 'lit-element';
+import {BaseStateElement} from '../BaseStateElement';
+import {store} from '../../store';
+import {setLanguage, checkUserPreferredLanguage} from '../../actions';
+
+const languageNames = {
+  en: 'English',
+  pl: 'Polish',
+};
+
+class LanguageSelect extends BaseStateElement {
+  static get properties() {
+    return {
+      userPreferredLanguage: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+    this.onStateChanged = this.onStateChanged.bind(this);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    checkUserPreferredLanguage();
+  }
+
+  onStateChanged() {
+    const state = store.getState();
+    this.userPreferredLanguage = state.userPreferredLanguage;
+  }
+
+  onSelect(e) {
+    setLanguage(e.target.value);
+  }
+
+  renderOption(language) {
+    return this.userPreferredLanguage === language
+      ? html`
+          <option value="${language}" selected>
+            ${languageNames[language]} (${language})
+          </option>
+        `
+      : html`
+          <option value="${language}">
+            ${languageNames[language]} (${language})
+          </option>
+        `;
+  }
+
+  render() {
+    return html`
+      <div class="w-display-flex">
+        <label class="w-visually-hidden" for="preferred-language">
+          Choose language
+        </label>
+        <select id="preferred-language" @change=${this.onSelect}>
+          ${['en', 'pl'].map((language) => this.renderOption(language))}
```

Comment:
If we go with a language constants file then we should use it for this array as well.